web/src/widgetize.ts

```diff
@@ -12,9 +12,11 @@ type ZFormExtraData = {
     choices: {type: string; reply: string; long_name: string; short_name: string}[];
 };
 
+type ExtraData = PollWidgetExtraData | ZFormExtraData | null;
```

Comment:
`WidgetExtraData` is probably a better name for this, will adjust when merging.

Reply:
Done; might also be good to rename the poll widget thing to `PollWidgetExtraData` in its own module, so we don't need to rename that on import.